pkg/controller/installation/core_controller.go

```diff
@@ -244,10 +245,12 @@ type ReconcileInstallation struct {
 	clusterDomain        string
 }
 
+const TigeraCustom = "unsupported.operator.tigera.io/custom"
```

Comment:
This annotation needs a comment explaining what it's purpose is. "custom" is very vague, and doesn't make it obvious what setting this annotation does.

Ideally, the name of the annotation would describe it's purpose.